easybuild/easyconfigs/x/XML-Parser/XML-Parser-2.44_01-foss-2018a-Perl-5.26.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PerlModule'
+
+name = 'XML-Parser'
+version = '2.44_01'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://search.cpan.org/~toddr/XML-Parser-2.41/'
+description = """This is a Perl extension interface to James Clark's XML parser, expat."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
```

Comment:
You can remove this easyconfig, the existing `XML-Parser-2.44_01-GCCcore-6.4.0-Perl-5.26.1.eb` suffices?